sync/sync-impl/src/main/res/layout/activity_connect_sync.xml

```diff
@@ -34,21 +34,17 @@
         android:visibility="gone"
         tools:ignore="ContentDescription"/>
```

Comment:
Is it expected that this activity (and activity_login_sync) are always on dark mode?

Reply:
Yes, it is like that by design